### PR TITLE
fix reload

### PR DESCRIPTION
**Pull request by Tjorven-Liebe**
after a while of using your software, I regcognised, that your reload wasn't working that well. 
I guess that I've fixed it in 2 lines. If not teach me.

### CODE REVIEW

**Review by JonasDevDE — APPROVED** [review]

**Comment by Konicai** [comment]
I can't speak for Tim but I'm pretty sure this should target the development branch if it hasn't already been fixed there.

File: spigot/src/main/java/org/geysermc/floodgate/inject/spigot/SpigotInjector.java [hunk 1]
```diff
@@ -107,6 +107,8 @@ public void onAdd(Object object) {
     }
 
     public void injectClient(ChannelFuture future) {
+        if (future.channel().pipeline().get("floodgate-init")!=null)
```
Comment (rtm516) [inline spigot/src/main/java/org/geysermc/floodgate/inject/spigot/SpigotInjector.java:110]:
```suggestion
        if (future.channel().pipeline().get("floodgate-init") != null)
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Comment by Tim203** [comment]
Hi, the reason this PR has not been merged is because I think this doesn't actually solve the issue (only on the surface.)
Some form of reload support is planned, but I don't think this is the way.

**Comment by Tjorven-Liebe** [comment]
Well if you think so.
I will close this PR now.

**Pull request closed without merging**